react/src/components/Chat/EndpointLLMChatCard.tsx

```diff
@@ -93,37 +97,59 @@ const EndpointLLMChatCard: React.FC<EndpointLLMChatCardProps> = ({
 
   const submitId = useId();
 
+  const { agents } = useAgents();
+  const [agentId, setAgentId] = useState(defaultAgentId);
+  const systemPrompt = useMemo(
+    () => agents.filter((a) => a.id === agentId)[0]?.config.system_prompt,
+    [agents, agentId],
+  );
+
   return (
     <LLMChatCard
       {...cardProps}
+      chatId={`${endpoint?.endpoint_id}_${agentId}`}
```

Comment:
The way to re-restart chat as agent has been changed. endpoint make change in url but agent is not. so I have to add chat id to useChat. Let me know any idea to solve it in the better way